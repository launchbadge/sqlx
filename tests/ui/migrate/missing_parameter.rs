fn main() {
    //Fails due to missing migration parameter 
    let _shaggy = "shaggy";
    sqlx::migrate!("../../../../tests/ui/migrate/migrations", parameters = [("my_user", "scooby"), ("fooby", _shaggy)]);
}
