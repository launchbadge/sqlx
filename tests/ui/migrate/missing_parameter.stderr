error: proc macro panicked
 --> tests/ui/migrate/missing_parameter.rs:4:5
  |
4 |     sqlx::migrate!("../../../../tests/ui/migrate/migrations", parameters = [("my_user", "scooby"), ("fooby", _shaggy)]);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Error processing parameters: migration 20250423195520_create_users.up was missing a parameter 'my_password' at line 3, column 50
  = note: this error originates in the macro `sqlx::migrate` (in Nightly builds, run with -Z macro-backtrace for more info)
